client/src/components/MeetingExperience.tsx

```diff
@@ -89,49 +88,54 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
     const locale = COMPOSITE_LOCALE_EN_US;
     const formFactorValue = new MobileDetect(window.navigator.userAgent).mobile() ? 'mobile' : 'desktop';
 
-    if (renderPostCall && postCall) {
-      return (
-        <Survey
-          postCall={postCall}
-          onRejoinCall={() => {
-            setRenderPostCall(false);
-            callWithChatAdapter.onStateChange((state: CallWithChatAdapterState) => {
-              //page is set to lobby to avoid flicker of Devices page when "rejoin call" is clicked
-              if (state.page === 'configuration') state.page = 'lobby';
-            });
-            callWithChatAdapter.joinCall();
-          }}
-        />
-      );
-    }
-
     return (
-      <CallWithChatComposite
-        adapter={callWithChatAdapter}
-        fluentTheme={fluentTheme}
-        options={{
-          callControls: {
-            chatButton: chatEnabled
-          }
-        }}
-        locale={{
-          component: locale.component,
-          strings: {
-            chat: locale.strings.chat,
-            call: {
-              ...locale.strings.call,
-              lobbyScreenWaitingToBeAdmittedTitle: waitingTitle,
-              lobbyScreenWaitingToBeAdmittedMoreDetails: waitingSubtitle
-            },
-            callWithChat: locale.strings.callWithChat
-          }
-        }}
-        icons={{
-          LobbyScreenWaitingToBeAdmitted: logo,
-          LobbyScreenConnectingToCall: logo
-        }}
-        formFactor={formFactorValue}
-      />
+      <>
+        {renderPostCall && postCall && (
+          <Survey
+            data-testid="Survey"
+            postCall={postCall}
+            onRejoinCall={async () => {
+              await callWithChatAdapter.joinCall();
+              setRenderPostCall(false);
+            }}
+          />
+        )}
+        <div
+          style={{
```

Comment:
I think we can separate this style properties to `MeetingExperience.styles.ts` file.

Reply:
Fixed this